kie-bom/pom.xml

```diff
@@ -157,6 +157,19 @@
         <classifier>sources</classifier>
       </dependency>
 
+      <!-- KIE Security -->
+      <dependency>
+        <groupId>org.drools</groupId>
```

Comment:
I believe that is fine. There are already other modules with `org.kie` groupId as well...